narwhal/primary/src/primary.rs

```diff
@@ -352,6 +352,9 @@ impl Primary {
             config.inbound_request_timeout_ms = Some(300_000);
             config.outbound_request_timeout_ms = Some(300_000);
             config.shutdown_idle_timeout_ms = Some(1_000);
+            config.connectivity_check_interval_ms = Some(2_000);
+            config.connection_backoff_ms = Some(5_000);
```

Comment:
Can we use a lower connection_backoff_ms, e.g. 100? We can revisit later too.

Reply:
Setting as low as `100ms` might be a bit aggressive . On the other side since we set the `connectivity_check_interval_ms` to 2 seconds, then we know that worst case scenario would be to retry a connection as soon as 2 seconds have passed - well probably a bit more since that depends on the evaluation. I would set this to `1 second` and see whether we could go lower